FunctionalTests/Microsoft.Bot.Builder.FunctionalTests/FacebookChatTests.cs

```diff
@@ -48,7 +48,7 @@ private async Task SendMessageAsync(string echoGuid)
                 request.Method = HttpMethod.Post;
                 request.RequestUri = new Uri(_botEndpoint);
 
-                await client.SendAsync(request);
+                var response = await client.SendAsync(request);
```

Comment:
Is 'response' used anywhere?

Reply:
Good point. That was another debug effort. Now removed.